### PR TITLE
[M] CANDLEPIN-872: Updated jira_target_version_check GH action

### PR DESCRIPTION
**Pull request by bakajstep**
- Fixed script in CI to check Jira target version and skip commits linked to Jira when the Jira card did not contain a SAT link to a bug.

### CODE REVIEW

**Comment by bakajstep** [comment]
as soon as this PR get approval i will backport the action to older branches

File: .github/scripts/jira_target_version_check.py
Comment (bakajstep) [reply .github/scripts/jira_target_version_check.py:62]: yes, i was checking this with @nikosmoum

**Review by joshmalbrecht — APPROVED** [review]

**Pull request merged**
